lib/common/common/src/zeros.rs

```diff
@@ -0,0 +1,17 @@
+use std::io::{Result, Write};
+
+static ZEROS: [u8; 8096] = [0u8; 8096];
```

Comment:
Out of curiosity, why 8k and not 4k?

Is it because of this IO constant? https://github.com/rust-lang/rust/blob/ec239b888f3945a6ce043db639f1741d14f08874/library/std/src/sys_common/io.rs#L3

Reply:
It's a coincidence, tho it's a nice observation.

My considerations:
- I've increased it a bit for possible future use. (right now we don't need more than 4k).
- It's a static array, its size contributes to the resulting on-disk binary size as it goes to the [`.data` section](https://en.wikipedia.org/wiki/Data_segment). So it' shouldn't be too big.
- A single module with a single 8k buffer is not more wasteful than a bunch of 4k buffers in every module that wants to write zeros.